Mathlib/Control/LawfulFix.lean

```diff
@@ -225,13 +233,17 @@ variable [(x y : _) → OmegaCompletePartialOrder <| γ x y]
 
 open OmegaCompletePartialOrder.Chain
 
-theorem continuous_curry : Continuous <| monotoneCurry α β γ := fun c ↦ by
```

Comment:
Same here, you should keep the original lemma but tag it `@[deprecated ωScottContinuous_curry (since := "2024-08-26")]`

Reply:
Done, thanks.